src/routes/deposit.js

```diff
@@ -26,17 +25,17 @@ export default class Deposit extends React.Component {
 
     this.state = {
```

Comment:
Let's get rid of `this.state` in favor of observables.

```es6
@observable value = 0;
@observable selectedIdx = 0;
```

So no need to setState, you can just set it like a normal property (but better to do that inside method decorated with `@action` decorator)